empress/support_files/js/empress.js

```diff
@@ -1174,5 +1174,29 @@ define([
         this.drawTree();
     };
 
+    /**
+     * Set a callback when a the node menu is shown on screen
+     *
+     * The callback will receive a list of samples as the only argument. This
+     * is inteded to be used with emperor.
+     *
+     * @param {Function} callback Callback to execute.
+     */
+    Empress.prototype.setOnNodeMenuVisibleCallback = function (callback) {
+        this._events.selectedNodeMenu.visibleCallback = callback;
+    };
+
+    /**
+     * Set a callback when a the node menu is removed from the screen
+     *
+     * The callback will receive a list of samples as the only argument. This
```

Comment:
```suggestion
     * The callback will receive an array of samples as the only argument. This
```

(since js technically doesn't have lists)